content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -87,14 +91,14 @@ Invoice prepaid support services
 When support services are billed on a fixed price, an invoice can be created before any work is
 completed on the issue. In this case, a service product with the invoicing policy set to
 :guilabel:`Prepaid/Fixed Price` would be used, just like :ref:`the section above
```

Comment:
This paragraph is referencing the "Invoicing Policy" field without directing the user to the service product, so a :guilabel: does not apply here
```suggestion
completed on the issue. In this case, a service product with the *Invoicing Policy* set to
*Prepaid/Fixed Price* would be used, just like :ref:`the section above
```